packages/office-ui-fabric-react/src/utilities/positioning/positioning.ts

```diff
@@ -263,6 +263,32 @@ function _flipToFit(
   };
 }
 
+/**
+ * Flips only the alignment edge of an element rectangle. This is used instead of nudging the alignment edges into position,
+ * when alignPerfectlyWithTarget is specified.
+ * Returns the initial elementEstimate if the flipping does not work.
+ * @param elementEstimate
+ * @param target
+ * @param bounding
+ * @param gap
+ */
+function _flipAlignmentEdgeOnly(elementEstimate: IElementPosition, target: Rectangle, bounding: Rectangle, gap: number): IElementPosition {
```

Comment:
I'd drop the "Only" you can just call it `_flipAlignmentEdge`. That keeps it a bit cleaner and makes it clear that this method really has one job and that's just the swap alignment edges.